dist/lob-api-bundled.yml

```diff
@@ -3086,277 +3083,7 @@ tags:
 
 
       ### Changelog
```

Comment:
will this title still show? will the section by empty?

Reply:
Yup, it will show Changelog as the header and below it will be a link to the beamer changelog